FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -195,6 +197,17 @@ static uint16_t pusOpenUdpPorts[ democonfigOPEN_UDP_PORTS_ARRAY_SIZE ];
  */
 static Connection_t pxEstablishedConnections[ democonfigESTABLISHED_CONNECTIONS_ARRAY_SIZE ];
 
+/**
+ * @brief Task status array which will store status information of tasks
```

Comment:
1. This grammar is awkward, please reword: _Array of task statuses, used to generate custom metrics._
2. `pxTaskList` is not a good name for this since it is not a list of tasks.  Please rename this, e.g. `pxTaskStatuses` or `pxTaskStatusList`.